data_model/src/account.rs

```diff
@@ -60,7 +63,23 @@ model! {
     ///
     /// let id = "user@company".parse::<AccountId>().expect("Valid");
     /// ```
-    #[derive(Debug, Display, Clone, PartialEq, Eq, PartialOrd, Ord, Hash, Constructor, Getters, Decode, Encode, DeserializeFromStr, SerializeDisplay, IntoSchema)]
+    #[derive(
```

Comment:
Seems like we have an editor war on `derive`s.
@mversic's  PRs have them in one line, if I'm not mistaken. Maybe we should sign a peace treaty to have less noise in diffs?

Reply:
IMO, this is on `rustfmt`. That program is next to useless for imposing a consistent style, because it fails to remove tabs in my PRs. And here the threshold for when to break up derives is set either too low or too high. 

The **solution** to the problem is to perform a self-review on the draft and ensure that any change that you commit is necessary. I can tell @mversic does that because he often leaves comments on his own PRs and I think it's a good practice.

Reply:
I'm on the side of `rustfmt`. We can configure it as we wish, but let's keep the formating it provides

Reply:
If formatting is not enforced by `rustfmt` it's not enforced. With that said, the whole point of moving to an attribute macro was to allow for tooling to inspect the internals of the module, and hence format. 

With that said, https://lkml.org/lkml/2020/5/29/1038, the 80 char limit isn't as uniformly accepted as it used to be.

Reply:
That's exactly what I said, no? 

> With that said, the whole point of moving to an attribute macro was to allow for tooling to inspect the internals of the module, and hence format.